lib/load.php

```diff
@@ -101,17 +101,17 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.0/class-wp-rest-block-patterns-controller.php';
 require __DIR__ . '/compat/wordpress-6.0/class-wp-rest-block-pattern-categories-controller.php';
 require __DIR__ . '/compat/wordpress-6.0/functions.php';
-require __DIR__ . '/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php';
+require __DIR__ . '/compat/wordpress-6.0/class-wp-theme-json-resolver-6-0.php';
```

Comment:
🤦‍♀️ my error on resync. `require __DIR__ . '/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php';` should remain and is the reason why the tests are failing. Doh. Resolving now.